.github/workflows/phpcs_on_pull_request.yml

```diff
@@ -0,0 +1,18 @@
+on: pull_request
+name: Inspections
+jobs:
+  runPHPCSInspection:
+    name: Run PHPCS inspection
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Run PHPCS inspection
+      uses: docker://rtcamp/action-phpcs-code-review:v2.0.0
+      env:
+        SKIP_FOLDERS: "tests,.github"
+        VAULT_ADDR: ${{ secrets.VAULT_ADDR }}
+        VAULT_TOKEN: ${{ secrets.VAULT_TOKEN }}
+      with:
+        args: WordPress,WordPress-Core,WordPress-Docs
```

Comment:
`WordPress` includes both `WordPress-Core` and `WordPress-Docs`. Not need to mention other 2.